jaraco/modb/__init__.py

```diff
@@ -16,13 +19,13 @@ def _flatten(self, obj):
         return super(Pickler, self)._flatten(obj)
 
 
-class Unpickler(jsonpickle.unpickler.Unpickler):
+class Unpickler(jsonpickle.unpickler.Unpickler):  # type: ignore[misc] # jsonpickle/jsonpickle#441
     pass
 
 
-def encode(value):
+def encode(value: object) -> Any | None:
     return Pickler().flatten(value)
 
 
-def decode(value):
+def decode(value: object) -> Any | None:
```

Comment:
See https://github.com/python/typeshed/blob/main/CONTRIBUTING.md#the-any-trick for `-> Any | None`